frontend/package.json

```diff
@@ -11,7 +11,7 @@
     "format": "prettier --write .",
     "format:check": "prettier --check .",
     "lint": "eslint --max-warnings=0 .",
-    "test": "jest",
+    "test": "vitest run",
```

Comment:
run is not needed.
when invoked in a ci environment vitest switches to non watch mode automagically.